Essentials/src/main/resources/config.yml

```
@@ -666,6 +666,11 @@ jail-online-time: false
 # Set to 0 for no timeout.
 tpa-accept-cancellation: 120
 
+# The maximum amount of simultaneous tpa requests that can be pending for any given user.
+# Once at this threshold, any new tpa requests will bump the oldest tpa requests out of queue.
+# Defaults to 15.
+tpa-max-amount: 15
```

amount -> requests to be more descriptive.
```suggestion
tpa-max-requests: 15
```